Apps/AdminWebClient/src/appsettings.Development.json

```diff
@@ -1,41 +1,41 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "HealthGateway": "Debug"
-    }
-  },
-  "AllowedHosts": "*",
-  "AddCors": {
-    "WithOrigins": "http://localhost:5010"
```

Comment:
As per RocketChat, was this change for hot reloading and is Chrome fixed?  I don't think we should have to make changes to Chrome to get the app to run as we're downgrading Chrome for every site.

Reply:
correct, hot reload requires using http instead of https. The problem with chrome seems to be related to not storing the cookies when coming from SSO

Reply:
I think we should pull this change out of this PR.